api/src/pcapi/scripts/algolia_settings.py

```diff
@@ -0,0 +1,119 @@
+import json
+import logging
+import os
+
+from algoliasearch.search_client import SearchClient
+from algoliasearch.search_index import SearchIndex
+import click
+
+from pcapi import settings as pcapi_settings
+from pcapi.utils.blueprint import Blueprint
+
+
+ALGOLIA_SETTINGS_DIR = os.path.abspath(
+    os.path.normpath(
+        os.path.join(
+            os.path.dirname(__file__),
+            os.path.pardir,
+        )
+    )
+)  # pass-culture-main/api/src/pcapi
+ALGOLIA_SETTINGS_DEFAULT_FILE = os.path.join(ALGOLIA_SETTINGS_DIR, "algolia_settings.json")
+LOGGER = logging.getLogger(__name__)
+
+
+blueprint = Blueprint(__name__, __name__)
+
+
+def _ensure_settings_dir_exists() -> None:
+    os.makedirs(ALGOLIA_SETTINGS_DIR, exist_ok=True)
```

Comment:
(détail / question de goût) En plus moderne, avec pathlib : 

```suggestion
    ALGOLIA_SETTINGS_DIR.mkdir(exist_ok=True)
```

Reply:
j'ai viré cette fonction qui ne sert pas vu qu'on stocke le fichier de settings dans `pcapi` qui, a prori, existe toujours.